### PR TITLE
Getting rid of centos_swan_image

### PR DESCRIPTION
**Pull request by iwankgb**
Fixes issue SCE-1156

Summary of changes:
- got rid of `centos_swan_image`

Testing done:
- all existing tests should pass.

### CODE REVIEW

**Comment by iwankgb** [comment]
run integration tests

File: vagrant/provision_ci_environment.sh [hunk 1]
```diff
@@ -46,32 +46,18 @@ if [ -e "$HOME_DIR/swan_s3_creds/.s3cfg" ]; then
...
-    echo "Synchronize /opt/swan"
-    # For manuall installtion
-    # sudo sh -c "mkdir -p /opt/swan && chown -R $USER:$USER /opt/swan"
-    s3cmd sync s3://swan-artifacts/workloads/swan/ /opt/swan/
+    echo "Download intelsdi/swan docker image"
+    docker pull intelsdi/swan
 
-    echo "Install binaries from /opt/swan"
-    ln -sv ${SWAN_BIN}/* /bin/
+    echo "Install binaries to /opt/swan and symlink them to /bin"
```
Comment (skonefal) [inline vagrant/provision_ci_environment.sh]: FYI: in #656 this is part of `provision_exp_environment`.

**Comment by iwankgb** [comment]
run integration tests

**Comment by skonefal** [comment]
@iwankgb kill docs/install.md

**Comment by iwankgb** [comment]
run integration tests

File: vagrant/provision_experiment_environment.sh [hunk 2]
```diff
@@ -141,6 +141,14 @@ wget --no-verbose https://github.com/intelsdi-x/snap-plugin-publisher-file/relea
 wget --no-verbose https://github.com/intelsdi-x/snap-plugin-collector-use/releases/download/${SNAP_PLUGIN_COLLECTOR_USE_VERSION}/snap-plugin-collector-use_linux_x86_64 -O ${SWAN_BIN}/snap-plugin-collector-use
 
 
+echo "----------------------------- Pulling docker image (`date`)"
+docker pull intelsdi/swan
```
Comment (skonefal) [inline vagrant/provision_experiment_environment.sh]: Both of 'docker pull' and 'retrieve binaries' are already in line 103. Please remove.

**Comment by iwankgb** [comment]
run integration tests

**Pull request merged**